Project1.md

```diff
@@ -22,9 +22,6 @@ running SSH, looking at Apache log files, etc. While you don't need to
 be a shell scripting god, you should also be able to get the gist of a
 script if need be.
 
-If you want a reference with additional commands, we're going to
-consistently update this document with common questions:
-https://docs.google.com/a/virginia.edu/document/d/1fCqyJQBS0Qoi0kWcS2B6xKGQHmUQzr-CpqEG1vQ5kak/edit?usp=sharing
```

Comment:
I have no idea why I made this a Google doc rather than putting it in git. If it's still of any value, here it is in markdown https://github.com/thomaspinckney3/cs4501/pull/72